bigquery/nox.py

```diff
@@ -93,6 +93,9 @@ def system(session, py):
     # Set the virtualenv dirname.
     session.virtualenv_dirname = 'sys-' + py
 
+    # Use pre-release gRPC for system tests.
+    session.install('--pre', 'grpcio')
```

Comment:
BigQuery doesn't have any dependencies on gRPC yet AFAIK. We use the REST API directly with Requests.